### DISCUSSION TITLE
I can't see the Japanese translation site.

### DISCUSSION THREAD

**Discussion by AkifumiSato** [react-hook-form/react-hook-form#9207] [Q&A]
Recently, I have been unable to view the Japanese translation of the site.
Has the maintenance of the Japanese translation stopped?
Or is it some kind of trouble?

e.g.
https://react-hook-form.com/jp/advanced-usage

**Comment by bluebill1049** [comment]
We haven't got enough time or team effort to follow up with all the translations. I am quite exhausted just to keep updating the main doc, here is page: https://legacy.react-hook-form.com/jp/advanced-usage Also try to figure out how the redirect works with gatsby, which having diffcult to get it working correctly as well...https://github.com/react-hook-form/documentation/pull/843
